uninstall.php

```diff
@@ -11,4 +11,34 @@
 	exit;
 }
 
-delete_option( 'perflab_modules_settings' );
+if ( is_multisite() ) {
+	delete_data_multisite();
+} else {
+	delete_data();
+}
+
+/**
+ * Delete data for the current blog.
+ */
+function delete_data() {
+
+	delete_option( 'perflab_modules_settings' );
+}
+
+/**
+ * Delete data for multisite blogs.
+ */
+function delete_data_multisite() {
+	global $wpdb;
+
+	$blog_ids         = $wpdb->get_col( "SELECT blog_id FROM {$wpdb->blogs}" );
```

Comment:
If we still go with SQL query approach then use `$wpdb->prepare` inside `$wpdb->get_col`. Example. https://developer.wordpress.org/reference/classes/wpdb/get_col/#highlighter_403812

Reply:
Thank you, changed it to `get_sites()`